bigbluebutton-html5/imports/ui/components/logout-confirmation/component.jsx

```diff
@@ -49,6 +74,8 @@ class LeaveConfirmation extends Component {
         }}
       >
         {intl.formatMessage(intlMessages.message)}
+        {this.renderButtonEndMeeting()}
```

Comment:
Please, prevent to have more than one render in each Component, make this EndMeetingButton a new Component if needed.

Reply:
I don't agree. The concept is right - it's easier to read when the code is split into logical parts. No reason for not doing it in the main render, especially if it returns a lot of stuff.
But that's not the case here, this main render() doesn't do a lot, I would just insert the button right here.